core/src/main/java/io/questdb/griffin/SqlCodeGenerator.java

```diff
@@ -1680,6 +1688,61 @@ private ObjList<Function> generateCastFunctions(
         return castFunctions;
     }
 
+    private RecordCursorFactory generateFill(QueryModel model, RecordCursorFactory groupByFactory, SqlExecutionContext executionContext) throws SqlException {
+        if (!isFillStridePresent(model.getNestedModel())) {
+            return groupByFactory;
+        } else {
+            try {
+                final QueryModel nested = model.getNestedModel();
+                Function fillFromFunc = TimestampConstant.NULL;
+                Function fillToFunc = TimestampConstant.NULL;
+                final ExpressionNode fillFrom = nested.getFillFrom();
+                final ExpressionNode fillTo = nested.getFillTo();
+                final ExpressionNode fillStride = nested.getFillStride();
+
+                ObjList<ExpressionNode> fillValuesExprs = nested.getFillValue();
+                ObjList<Function> fillValues = new ObjList<>(fillValuesExprs.size());
```

Comment:
Don't we need a null check somewhere? What prevents `fillValuesExprs` from being null?